lms/static/sass/elements/_program-card.scss

```diff
@@ -72,15 +110,32 @@ $card-height: 150px;
   .program-card{
     @include omega(2n);
     @include span-columns(6);
-    display:inline;
+    .card-link{
+      .banner-image-container{
+        height:116px;
+      }
+    }
+    .text-section{
+      margin-top:106px;
+      .title{
+        font-size:em(24);
+      }
+    }
   }
 }
 
 @include media($bp-huge) {
   .program-card{
     @include omega(2n);
     @include span-columns(6);
-    display:inline;
+    .card-link{
+      .banner-image-container{
+        height:145px;
+      }
+    }
+    .text-section{
+      margin-top:135px;
+    }
   }
 }
 
```

Comment:
Nitpick: the formatting for this file is inconsistent. Can you please update so there is always a single space between the  colon and the value (e.g. ```height: 145px;```).